packages/playground/src/components/k8s_deployment_table.vue

```diff
@@ -85,9 +84,15 @@
       </template>
       <template #[`item.status`]="{ item }">
         <v-chip :color="getNodeHealthColor(item.value.masters[0].status as string).color">
-          <v-tooltip v-if="item.value.masters[0].status == NodeHealth.Error" activator="parent" location="top">{{
-            item.value.masters[0].message
-          }}</v-tooltip>
+          <v-tooltip
+            v-if="item.value.masters[0].status == NodeHealth.Error || item.value.masters[0].status == NodeHealth.Paused"
+            activator="parent"
+            location="top"
+            >{{ item.value.masters[0].message }}</v-tooltip
+          >
+          <v-tooltip v-if="item.value.masters[0].status == NodeHealth.Paused" activator="parent" location="top"
+            >This node contract is in the grace period</v-tooltip
```

Comment:
what do u think about changing it to `The deployment contract is in grace period`.

Reply:
message updated